src/nupic/utils/StringUtils.cpp

```diff
@@ -81,6 +81,24 @@ Real32 StringUtils::toReal32(const std::string& s, bool throwOnError, bool * fai
   return r;
 }
 
+Real64 StringUtils::toReal64(const std::string& s, bool throwOnError, bool * fail) {
```

Comment:
I'd drop the bool fail, bool throwOnErr logic, and really throw an exception if conversion fails. The caller must handle that. Exceptions are proper way to send this kind of signals

Reply:
> and really throw an exception if conversion fails.

This was actually a little outside this PR's objectives.  I just noticed that toReal64 was missing from the set of other conversion tools and I just copied one of the others and changed the type.  If we want exceptions we should do this for all of them in a different PR.   But frankly, there are times that a blank field is not an error and it makes since to do it this way.